### PR TITLE
Update command style in serving and admin directories

### LINKED ISSUE #3813: Split kubectl commands using <<EOF into two steps
The current method for documenting `kubectl` commands YAML using `kubectl apply -f - <<EOF...` does not work for Windows users.

To fix this issue, do the following:

- [x] Update style rule in the [Documenting YAML](https://dev-knative.netlify.app/development/help/contributor/style-guide/documenting-code/) section of the style guide. File location: https://github.com/knative/docs/blob/mkdocs/docs/help/contributor/style-guide/documenting-code.md

     The new steps should be similar to the following:

   >1. Create a YAML file...
   >     ```
   >     # YAML FILE CONTENTS
   >     ```
   > 2. Apply the YAML file...
   >     ```
   >     kubectl apply --filename <filename>.yaml
   >     ```

- [x] Search the knative/docs repo for kubectl commands that use the `<<EOF` format. Update these commands to follow the new style rules.

     - Update all commands in the docs
     - Optional: Update commands in `tutorial` and `blog` folders 

See this [Slack thread](https://knative.slack.com/archives/C9CV04DNJ/p1623783139041200) for the discussion.

### PR DESCRIPTION
**Pull request by snneji**
Update the commands from using `<<EOF` format to a two step procedure. 

Lots of files to change for the second part of #3813 so this PR will only change the files in the `/admin` and `/serving` directories.

### CODE REVIEW

**Comment by abrennan89** [comment]
Careful writing "Fixes #3813." since this PR is now linked to that issue and will close it. You might want to unlink it or you'll probably need to reopen the issue again later.

File: docs/admin/eventing/broker-configuration.md [hunk 1]
```diff
@@ -112,15 +112,23 @@ When a Broker is created without a specified `BrokerClass` annotation, the defau
...
+    kind: Broker
+    metadata:
+      name: default
+      namespace: default
+    ```
+
+1. Apply the YAML file by running the command:
+
+    ```bash
+    kubectl apply --filename <filename>.yaml
```
Comment (abrennan89) [inline docs/admin/eventing/broker-configuration.md:128]:
```suggestion
    kubectl apply -f <filename>.yaml
```
I'd prefer we use just `-f` across the board and shorten the commands generally where possible. We'd need to update it in the style guide though (https://dev-knative.netlify.app/development/help/contributor/style-guide/documenting-code/#documenting-yaml)

Wdyt @RichardJJG @snneji ?

File: docs/serving/using-auto-tls.md [hunk 2]
```diff
@@ -45,72 +45,75 @@ Knative supports the following Auto TLS modes:
...
+        - See the generic [`ClusterIssuer` example](https://cert-manager.io/docs/configuration/acme/#creating-a-basic-acme-issuer)
+        - Also see the
+        [`DNS01` example](https://docs.cert-manager.io/en/latest/tasks/acme/configuring-dns01/index.html)
+
+        For example, the following `ClusterIssuer` file named `letsencrypt-issuer` is
+        configured for the Let's Encrypt CA and Google Cloud DNS.
+        The Let's Encrypt account info, required `DNS-01` challenge type, and
+        Cloud DNS provider info is defined under `spec`.
+
+        ```bash
```
Comment (abrennan89) [inline docs/serving/using-auto-tls.md:63]:
```suggestion
        ```yaml
```

**Review by abrennan89 — APPROVED** [review]
Couple of minor comments.

In general, I'd prefer we use `-f` than writing out `--filename` for every command, and update the style guide. Wdyt @RichardJJG @snneji ?

File: docs/admin/eventing/broker-configuration.md (hunk 1 above)
Comment (snneji) [reply docs/admin/eventing/broker-configuration.md:128]: I agree. I can make those changes.

**Comment by RichardJJG** [comment]
> Couple of minor comments.
> 
> In general, I'd prefer we use `-f` than writing out `--filename` for every command, and update the style guide. Wdyt @RichardJJG @snneji ?

Yep, I'm fine with that and I can see Samia made the change.

**Comment by snneji** [comment]
> Yep, I'm fine with shortening the flag to `-f` if that's permitted. And you think that should be a style rule, @abrennan89? Can we confidently tell all writers and engineers to go with that in all cases?

Yes.  `-f` is the same as `--filename`. I've updated the style guide here https://github.com/knative/docs/pull/3958.

**Comment by snneji** [comment]
@abrennan89 I've updated the commands to use `-f` instead of `--filename`. I've also updated the style guide. Is this PR good to merge now?

**Review by abrennan89 — APPROVED** [review]
/lgtm

Yep, looks good @snneji !
The preview isn't working for me atm so I can't check that, but if there's anything else we can follow up another time. Thanks.

**Comment by snneji** [comment]
@abrennan89 can I remove the hold label?

**Review by abrennan89 — APPROVED** [review]
/lgtm
/unhold

**Pull request merged**